<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerQt</class>
 <widget class="QWidget" name="ServerQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerQt</string>
  </property>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="textHost">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ip: 172.0.0.1 port: 5002</string>
   </property>
  </widget>
  <widget class="QLabel" name="sendMsg">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>not message</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
